### ISSUE TITLE
Make jquery releases be brotli compressed by default

### ISSUE THREAD

**Issue by LifeIsStrange** [jquery/jquery#5093]
It allows for smaller size than gzip, and is universally supported in 2022
https://caniuse.com/brotli
Note that this would be effective after dropping IE support

**Comment by mgol** [comment]
Thanks for the report. This one is already tracked at https://github.com/jquery/codeorigin.jquery.com/issues/62 (this is the repo where the source for code.jquery.com & releases.jquery.com lies). AFAIK we’re still blocked by Stackpath not supporting Brotli natively but requiring some workarounds.

Also, we’re in the process of a big CDN setup migration right now so any improvements will most likely need to wait until that’s finalized.

**Comment by LifeIsStrange** [comment]
I'm pretty sure being ~the most used js library out there would allow you to put enough pressure on Stackpath to properly support brotli in order to not become irrelevant

**Comment by mgol** [comment]
At one point I’d like to return to the open issues around CDN setup (there are more of them) but this will need to wait until our current migration is finished and until someone has time to fully dive into the issue, ping relevant people, etc.
